CommonData/filter.cpp

```diff
@@ -3,6 +3,8 @@
 #include "dataset.h"
 #include "databaseinterface.h"
 
+std::map<std::string, Filter*>	_filterMap;
```

Comment:
This is a terrible Idea, a global really?

Reply:
OK, I have changed this